engine/src/main/java/org/terasology/rendering/dag/nodes/InitialPostProcessingNode.java

```diff
@@ -112,7 +99,7 @@ public InitialPostProcessingNode(Context context) {
         if (renderingConfig.isLightShafts()) {
             addDesiredStateChange(new SetInputTextureFromFBO(textureSlot++, LIGHT_SHAFTS_FBO, ColorTexture, displayResolutionDependentFBOs, INITIAL_POST_MATERIAL, "texLightShafts"));
         }
-        addDesiredStateChange(new SetInputTexture(textureSlot++, Assets.getTexture("engine:vignette").get().getId(), INITIAL_POST_MATERIAL, "texVignette"));
+        addDesiredStateChange(new SetInputTexture(textureSlot, Assets.getTexture("engine:vignette").get().getId(), INITIAL_POST_MATERIAL, "texVignette"));
```

Comment:
In line with previous comments, let's create a getTextureId(string or SimpleUri) method.